docs/getting-started.md

```diff
@@ -242,6 +242,33 @@ curl --unix-socket /tmp/firecracker.socket -i  \
     }'
 ```
 
+#### Configuring the microVM without sending API requests
+
+If you'd like to boot up a guest machine without using the API socket, you can do that 
+by passing the parameter `vmm-config` to the Firecracker process. The command for starting 
+Firecracker with this option will look like this:
+
+```bash
+./firecracker --api-sock /tmp/firecracker.socket --vmm-config "JSON_BODY"
+```    
+
+"JSON_BODY" should contain the entire configuration for all of the microVM's resources. 
+You **must** provide in your JSON the configuration for the guest kernel and rootfs, as 
+these are mandatory, but all of the other resources are optional, so it's your choice if 
```

Comment:
Clearly specify that using this configuration method will also start the microVM with said configuration, so that all desired pre-boot configurable resources need to be specified.